src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -24,7 +24,7 @@
     final protected static Logger logger = LogManager.getLogger(DefaultDocWorkUnitHandler.class);
 
     private static final String NAME_FOR_POSITIONAL_ARGS = "[NA - Positional]";
-    private static final String DEFAULT_FREEMARKER_TEMPLATE_NAME = "generic.template.html";
+    private static final String DEFAULT_FREEMARKER_TEMPLATE_NAME = "generic.ftl";
```

Comment:
I think it would be useful to retain the ".html" in the file name to indicate what kind of template these are, maybe "generic.html.ftl".